packages/react-adminer/src/components/SectionBox.tsx

```diff
@@ -140,6 +141,11 @@ const SectionBox: React.FC<Props> = ({
 				);
 			})}
 		</Row>
+		{!sectionFields.some(f => f.creatable === true) && (
```

Comment:
Vytahni to do promenne, tu pojmenuj tak aby slo poznat co to je za stav a tu pouzij. Takhle uplne neni videt o co jde.